src/components/TaskStatusBadge.tsx

```diff
@@ -72,46 +74,52 @@ const BadgeWidthMaxContent = styled(Badge)`
 interface TaskStatusBadgeProps {
   status: string;
   blocked: boolean;
+  aborted: boolean;
 }
 export const TaskStatusBadge: React.FC<TaskStatusBadgeProps> = ({
   status,
   blocked,
+  aborted,
 }) => {
-  // We have to do this assignment because Blocked is not an official Task
-  // status from the tasks collection but we want to represent it in the badge.
-  const adjustedTaskStatus = blocked
-    ? (TaskStatus.StatusBlocked as string)
-    : status;
+  // We have to do this assignment because Blocked and Aborted are not official Task
+  // statuses from the tasks collection but we want to represent it in the badge.
+  let taskStatus = status;
```

Comment:
a switch statement could make the assignment logic easier to understand

Reply:
if there were more than two if statements then sure. but i think it's fine and readable as is.